alacritty_terminal/src/term/mod.rs

```diff
@@ -40,8 +40,13 @@ const MAX_SEARCH_LINES: usize = 100;
 /// Default tab interval, corresponding to terminfo `it` value.
 const INITIAL_TABSTOPS: usize = 8;
 
-/// Minimum number of columns and lines.
-const MIN_SIZE: usize = 2;
+/// Minimum number of columns.
+///
+/// A minimum of 2 is necessary to hold fullwidth unicode characters.
+pub const MIN_COLS: usize = 2;
```

Comment:
Let's name those variables consistenlty, so it should likely be `MIN_SCREEN_COLS`.

Reply:
No, there is no need to call these "screen cols" because there's no difference between screen and absolute columns.

This is consistent within our codebase.

Reply:
We indeed call them differently due to scrollback, etc.